libs/model-ad/model-details/src/lib/components/model-details-boxplots-selector/model-details-boxplots-selector.component.ts

```diff
@@ -90,14 +115,74 @@ export class ModelDetailsBoxplotsSelectorComponent {
     );
   }
 
+  getDefaultTissue() {
+    return this.tissueOptions()[0] || '';
+  }
+
+  initializeOptionsFromUrlParams() {
+    const sexParam = this.helperService.getUrlParam(this.SEX_QUERY_KEY);
+    const tissueParam = this.helperService.getUrlParam(this.TISSUE_QUERY_KEY);
+
+    const matchingSexOption = this.sexOptions.find((option) => option.label === sexParam);
+    if (matchingSexOption !== undefined) this.selectedSexOption.set(matchingSexOption);
+
+    const matchingTissueOption = this.tissueOptions().find((option) => option === tissueParam);
+    this.selectedTissueOption.set(matchingTissueOption || this.getDefaultTissue());
+
+    this.hasInitializedOptions = true;
+  }
+
+  initialScrollToSection() {
+    if (typeof window !== 'undefined' && !this.isInitialScrollDone) {
+      const hash = window.location.hash.slice(1);
```

Comment:
for clarity, consider adding a comment explaining the specific value of the hash variable like:
// hash variable contains everything after the # in the URL and slice(1) removes the leading # character